packages/myst-cli/src/build/site/watch.ts

```diff
@@ -164,14 +172,16 @@ export function watchContent(session: ISession, serverReload: () => void, opts:
         ? localProjects.filter(({ path }) => path !== '.').map(({ path }) => join(path, '*'))
         : [];
     if (siteConfigFile) ignored.push(siteConfigFile);
+    const projectConfig = selectors.selectLocalProjectConfig(state, proj.path);
+    if (projectConfig?.exclude) ignored.push(...projectConfig.exclude);
     const dependencies = new Set(selectors.selectAllDependencies(state, proj.path));
     chokidar
       .watch([proj.path, ...dependencies], {
         ignoreInitial: true,
         ignored: ['public', '**/_build/**', '**/.git/**', ...ignored],
```

Comment:
There is an edge case here:

When you update `exclude` in the project config, it will rebuild the project correctly with the new `exclude` value respected. However, `chokidar.watch` is not updated so if you stop excluding a file you will not stop watching it, and if you exclude a previously watched file, it will continue to be watched. I don't think this is a big deal... worst case scenario is you just kill your `myst start` process and call it again to re-watch the correct stuff.  (And if it is a problem, we can probably fix it....)

Reply:
Yep, I think it shouldn't be that big a deal that the watch is lagging.